.github/workflows/push-to-master.yml

```diff
@@ -62,6 +62,8 @@ jobs:
           python -m pytest modin/numpy/test/test_array_logic.py
           python -m pytest modin/numpy/test/test_array_linalg.py
           python -m pytest modin/numpy/test/test_array_indexing.py
+          python -m pytest modin/numpy/test/test_array_math.py
```

Comment:
Does it make sense to do so? (so far we don't have many `numpy` tests): `python -m pytest modin/numpy/test`?
This will increase the probability that we will remember to add testing new files to all workloads.

Reply:
Ok, let's solve the question separately.